docs/src/main/sphinx/release/release-444.md

```diff
@@ -0,0 +1,102 @@
+# Release 444 (3 Apr 2024)
+
+## General
+
+* Improve planning time for queries with large number of joins. ({issue}`21360`)
+* Fix failure for queries containing large numbers of `LIKE` terms in boolean
+  expressions. ({issue}`21235`)
+
+## Docker image
+
+* Update Java runtime to Java 22. ({issue}`21161`)
+
+## BigQuery connector
+
+* Fix failure when reading BigQuery views with [Apache
+  Arrow](https://arrow.apache.org/docs/). ({issue}`21337`)
+
+## ClickHouse connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## Delta Lake connector
+
+* Add support for reading `BYTE_STREAM_SPLIT` encoding in Parquet files. ({issue}`8357`)
```

Comment:
This is too low level. What's the user visible impact? What types does it allow reading now?